integration_tests/seeds/sf_lead_history_data.csv

```diff
@@ -0,0 +1,2 @@
+_fivetran_active,_fivetran_start,_fivetran_end,country,email_bounced_reason,email_bounced_date,owner_id,lead_source,converted_date,last_modified_date,master_record_id,last_modified_by_id,geocode_accuracy,converted_contact_id,id,state,last_referenced_date,last_activity_date,phone,name,created_by_id,is_converted,is_unread_by_owner,status,city,latitude,industry,title,_fivetran_synced,last_viewed_date,is_deleted,street,company,first_name,email,website,last_name,number_of_employees,created_date,postal_code,has_opted_out_of_email,converted_account_id,mobile_phone,annual_revenue,description,individual_id
```

Comment:
Let's remove this since we are not using it in the end models anymore.

Reply:
Removed